unified_planning/model/expression.py

```diff
@@ -275,6 +276,18 @@ def FluentExp(
             node_type=OperatorKind.FLUENT_EXP, args=tuple(params_exp), payload=fluent
         )
 
+    def Dot(
+        self, agent: "up.model.Agent", fluent_exp: List["up.model.fnode.FNode"]
```

Comment:
Why a **list** of `FNode`? You are representing a term like `a.f` where `a` is an agent and `f` is a fluent exp, so a single `FNode` should suffice, no?

Reply:
Done!